components/Notifications/NotificationEntry.js

```diff
@@ -320,6 +320,23 @@ const NotificationEntry = (props) => {
         );
       case "purchase": // synanomous to "support"
         return renderContentSupportNotification();
+      case "bounty":
+        const expirationTimeStamp = (
+          <TimeStamp date={action_item.expiration_date} timeSince={true} />
+        );
+        return (
+          <div className={css(styles.message)}>
+            {" Your bounty is expiring soon: "}
```

Comment:
Can we change to:

You bounty is expiring in **24 hours**. Please award it to the best answer.

Reply:
Updated